internal/feature/onboarding/src/main/kotlin/com/recco/internal/feature/onboarding/OnboardingScreen.kt

```diff
@@ -58,21 +57,23 @@ private fun OnboardingScreen(
 ) {
     val pages = listOf<@Composable (PaddingValues) -> Unit>({
         OnboardingPage(
-            imageId = drawable.recco_ic_flying,
+            image = { modifier -> AppTintedImageFlying(modifier) },
             titleTextId = string.recco_onboarding_page1_title,
             bodyTextId = string.recco_onboarding_page1_body,
             contentPadding = it
         )
     }, {
         OnboardingPage(
-            imageId = drawable.recco_ic_content,
+            image = { modifier -> AppTintedImageContent(modifier) },
             titleTextId = string.recco_onboarding_page2_title,
             bodyTextId = string.recco_onboarding_page2_body,
             contentPadding = it
         )
     }, {
         OnboardingPage(
-            imageId = drawable.recco_ic_portrait_2,
+            image = { modifier ->
+                AppTintedImagePortrait(modifier)
```

Comment:
why this statement is split in multiple lines when all previous are single line statements ?

Reply:
No reason, should spotless precisely fix this kind of issues?

Reply:
I guess that depends on how do we want things to be, depending on our preferences I mean

Reply:
I changed it anyway 👍